docs/source/conf.py

```diff
@@ -30,18 +30,19 @@ def setup(app):
 
 html_theme_options = {
     "logo": "logo_small.png",
+    "body_text_align": "left",
```

Comment:
Any particular reason to move these (`"body_text_align"` and `"donate_url"`?

Reply:
No, I think that changed while I was trying to work with badge_branch, I will reset this change.